src/plugins/intel_cpu/src/nodes/mkldnn_mvn_node.cpp

```diff
@@ -794,94 +918,83 @@ void MKLDNNMVNNode::prepareParams() {
     const SizeVector in_dims = srcMemPtr->getStaticDims();
     transformTo5DCase(in_dims);
 
-    setPostOps(attr, true);
-
     if (mayiuse(cpu::x64::sse41)) {
         auto selectedPD = getSelectedPrimitiveDescriptor();
-        auto jcp = jit_mvn_config_params();
-        jcp.src_prc = selectedPD->getConfig().inConfs[0].desc->getPrecision();
-        jcp.dst_prc = selectedPD->getConfig().outConfs[0].desc->getPrecision();
-        jcp.src_data_size = MKLDNNExtensionUtils::sizeOfDataType(MKLDNNExtensionUtils::IEPrecisionToDataType(jcp.src_prc));
-        jcp.dst_data_size = MKLDNNExtensionUtils::sizeOfDataType(MKLDNNExtensionUtils::IEPrecisionToDataType(jcp.dst_prc));
-        jcp.planar_layout = selectedPD->getConfig().inConfs[0].desc->hasLayoutType(LayoutType::ncsp);
-        jcp.normalize_variance = normalizeVariance_;
-        jcp.across_channels = execAcrossChannels_;
-        int N = 0;
-        std::tie(N, jcp.C, jcp.D, jcp.H, jcp.W) = shape5D;
-
-        if (mayiuse(cpu::x64::avx512_common)) {
-            mvn_kernel.reset(new jit_uni_mvn_kernel_f32<cpu::x64::avx512_common>(jcp, *attr.get()));
-
-            jcp.normalize_variance = false;
-            mvn_mean_kernel.reset(new jit_uni_mvn_mean_variance_kernel_f32<cpu::x64::avx512_common>(jcp));
-            if (normalizeVariance_) {
-                jcp.normalize_variance = true;
-                mvn_variance_kernel.reset(new jit_uni_mvn_mean_variance_kernel_f32<cpu::x64::avx512_common>(jcp));
-            }
-        } else if (mayiuse(cpu::x64::avx2)) {
-            mvn_kernel.reset(new jit_uni_mvn_kernel_f32<cpu::x64::avx2>(jcp, *attr.get()));
-
-            jcp.normalize_variance = false;
-            mvn_mean_kernel.reset(new jit_uni_mvn_mean_variance_kernel_f32<cpu::x64::avx2>(jcp));
-            if (normalizeVariance_) {
-                jcp.normalize_variance = true;
-                mvn_variance_kernel.reset(new jit_uni_mvn_mean_variance_kernel_f32<cpu::x64::avx2>(jcp));
-            }
-        } else if (mayiuse(cpu::x64::sse41)) {
-            mvn_kernel.reset(new jit_uni_mvn_kernel_f32<cpu::x64::sse41>(jcp, *attr.get()));
-
-            jcp.normalize_variance = false;
-            mvn_mean_kernel.reset(new jit_uni_mvn_mean_variance_kernel_f32<cpu::x64::sse41>(jcp));
-            if (normalizeVariance_) {
-                jcp.normalize_variance = true;
-                mvn_variance_kernel.reset(new jit_uni_mvn_mean_variance_kernel_f32<cpu::x64::sse41>(jcp));
-            }
-        }
-
-        if (mvn_kernel)
-            mvn_kernel->create_ker();
+        mvnAttrs.src_prc = selectedPD->getConfig().inConfs[0].desc->getPrecision();
+        mvnAttrs.dst_prc = selectedPD->getConfig().outConfs[0].desc->getPrecision();
+        mvnAttrs.planar_layout = selectedPD->getConfig().inConfs[0].desc->hasLayoutType(LayoutType::ncsp);
+        mvnAttrs.is_nhwc = getParentEdgeAt(0)->getMemory().getDesc().hasLayoutType(LayoutType::nspc);
```

Comment:
create layout enum and have only one member in mvnAttrs like [interpolate](https://github.com/openvinotoolkit/openvino/pull/9524/files#diff-abd23aa18b5eca6d6e04d158f40f8acb145889e5db9dd5321012d671adcf0302R18)?

Reply:
Fixed in PR[9889](https://github.com/openvinotoolkit/openvino/pull/9889)